cmake_file_api/kinds/toolchains/v1.py

```diff
@@ -0,0 +1,126 @@
+from enum import Enum
+import json
+from pathlib import Path
+from typing import Dict, List
+
+from cmake_file_api.kinds.common import VersionMajorMinor
+from cmake_file_api.kinds.kind import ObjectKind
+
+
+class ToolchainLanguageType(Enum):
```

Comment:
I think hard coding all tool chains + compiler types in an enum is not maintainable.
Returning a string is just fine.